notebooks/Emukit-tutorial-bayesian-optimization-integrating-model-hyperparameters.ipynb

```diff
@@ -0,0 +1,385 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bayesian optimization integrating model hyper-parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we are going to see how to use Emukit to solve optimization problems the acquisition function is integrated with respect to the hyper-parameters of the model. \n",
+    "\n",
+    "To show this with an example, we minimize the use the Six-hump camel function \n",
```

Comment:
too many words: To show this with an example, we <del>minimize the</del> use the Six-hump camel function

Reply:
Corrected